tests/Functional/FunctionalTest.php

```diff
@@ -0,0 +1,79 @@
+<?php declare(strict_types=1);
+
+namespace Bref\SymfonyBridge\Test\Functional;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+class FunctionalTest extends TestCase
+{
+    public function test Symfony works with an empty cache(): void
+    {
+        $this->composerInstall();
+        $this->clearCache();
+        $symfonyConsole = $this->runSymfonyConsole();
+        $this->assertTrue($symfonyConsole->isSuccessful());
+    }
+
+    public function test Symfony works with a compiled container(): void
```

Comment:
spaces in method names? 🤨

Reply:
https://mnapoli.fr/using-non-breakable-spaces-in-test-method-names/ 😉 

Used in the tests in Bref, for example https://github.com/brefphp/bref/blob/master/tests/Functional/FpmRuntimeTest.php